mono/tests/Makefile.am

```diff
@@ -65,9 +95,58 @@ STRESS_TESTS_SRC=	\
 	process-stress.cs	\
 	assembly-load-stress.cs
 
+BASE_TEST_MOBILE_STATIC_NOT_SUPPORTED= \
```

Comment:
I lot of those tests don't look to include things not suppported by FullAOT.
Are some on this list because they are just broken but supported to work?